paper-calendar.html

```diff
@@ -679,10 +677,10 @@
         var i = ((this.currentYear * 12) + this.currentMonth) -
                  ((this._months[0].year * 12) + this._months[0].month);
         this._translateX(-i * this._containerWidth);
-      },
-      _updateSelection: function() {
+      }
+      _updateSelection() {
         // Force the day selection circle to maintain a 1:1 ratio
-        var selected = this.$$('.day-item.selected');
+        var selected = this.shadowRoot.querySelector('.day-item.selected');
```

Comment:
Have you tested this in both shadow and shady dom?  I can't remember the specifics for the change on this function off the top of my head, but I think it was using `this.root` instead of `this.shadowRoot`

Reply:
Not sure what you mean by that I have tested and it works